webviews/components/Home.svelte

```diff
@@ -1,4 +1,4 @@
 <script lang="ts">
 </script>
 
-<h1>Hello World!!</h1>
\ No newline at end of file
+<h1>Now it is working fine! Thanks guys!</h1>
```

Comment:
I can't seem to reproduce this. It's not showing "Hello World", but it's not showing this either.
It's completely blank.
Although the function "onSignIn" _is_ called (I used this PR to check my oauth token by console.logging inside "onSignIn" case and it did work)
Without this PR, when I tried to check my oauth token, I couldn't reach that function itself - so that's partly good news.

Reply:
<img width="1440" alt="Screen Shot 2021-02-20 at 1 17 56 PM" src="https://user-images.githubusercontent.com/46453883/108606166-162c2f00-737e-11eb-8954-53cae2779174.png">

Reply:
@aitikgupta Can you see this?  So, I did this by running the `Hello World` command.  

Okay so while these changes are much appreciated, let's just call it off for now on merging until we can get reproducibility for us all.  @ShrillShrestha I wanna know why did you put the ext_vscode variable declaration!  You basically beat me to it except mine was called tsvscode because I was just following those tutorials to the letter!  So, I want to know what is ext_vscode intended to represent?  

@ShrillShrestha, if there's anything in this PR you still want to merge, let's merge it onto the main branch!  @ShrillShrestha